core/Plugin/Dimension/ActionDimension.php

```diff
@@ -191,6 +191,23 @@ public function onNewAction(Request $request, Visitor $visitor, Action $action)
     }
 
     /**
+     * Returns `true` if the column for this dimension is a reference to the `log_action` table (ie, an "idaction column"),
+     * `false` if otherwise.
+     *
+     * @return bool
+     */
+    public function isActionReference()
```

Comment:
Sorry, my bad! I was just looking over all pull requests and didn't notice the missing needs review :) It would be somehow nice to have it somewhere public or protected to still have it testable. If it's indirectly covered by tests of this class then it could be okay to have it as a private method. I'm just scared someone might eg remove the `Exception` from `getActionId` and then this code will not work as intended.